src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js

```diff
@@ -34,11 +34,36 @@ export const renderPlaylistUnit = (unit, t, views = -1, watchingNow = -1, sugges
     <Table.Body>
       <Table.Row verticalAlign="middle">
         <Table.Cell textAlign="left" width={4}>
-          <UnitLogo
-            unitId={unit.id}
-            collectionId={getCollectionId(unit)}
-            fallbackImg='programs'
-          />
+          <div className="recommend-cell">
```

Comment:
Can we move this class to the above Table.Cell?

Reply:
No as the cell height might be much higher than the div.